src/graphics/program-lib/chunks/lightSpecularAnisoGGX.frag

```diff
@@ -0,0 +1,34 @@
+// Anisotropic GGX
+uniform float material_anisotropy;
+float getLightSpecular() {
+    float PI=3.141592653589793;//2384626433832795‬;
```

Comment:
You probably don't need to continue the value of Pi into a comment here. 😄

Reply:
I shall remove it